src/server/actions/__tests__/saveChapter.test.js

```diff
@@ -29,6 +29,22 @@ describe(testContext(__filename), function () {
     expect(newChapter.id).to.eq(values.id)
   })
 
+  it('validates that chapter name is unique', async function () {
+    const chapter = await factory.build('chapter')
+    const chapterWithSameName = await factory
+      .build('chapter', {name: chapter.name})
+    await saveChapter(chapter)
+    expect(saveChapter(chapterWithSameName))
+      .to.be.rejectedWith('Chapter name must be unique')
+  })
+
+  it('changes name to lower case and removes white spaces', async function () {
+    const chapter = await factory.build('chapter', {name: 'aBc dEf'})
+    await saveChapter(chapter)
+    const sameChapter = await getChapter(chapter.id)
+    expect(sameChapter.name).to.eq('abcdef')
```

Comment:
It's actually important that characters in chapter names retain their original case. So `Oakland` should be stored as `Oakland`, not `oakland`. You'd only want to transform this value to lowercase (or uppercase) for the purpose of checking uniqueness.